etc/bento.env

```diff
@@ -272,15 +274,12 @@ BENTOV2_FEDERATION_CPUS=2
 
 # Event-Relay
 BENTO_EVENT_RELAY_REPO=https://github.com/bento-platform/bento_event_relay
-BENTOV2_EVENT_RELAY_TAG_OR_BRANCH=v1.0.0
-BENTOV2_EVENT_RELAY_BASE_IMAGE=node
-BENTOV2_EVENT_RELAY_BASE_IMAGE_VERSION=15.5.1-alpine3.10
-BENTOV2_EVENT_RELAY_IMAGE=bentov2-event-relay
-BENTOV2_EVENT_RELAY_VERSION=0.0.1
+BENTOV2_EVENT_RELAY_IMAGE=ghcr.io/bento-platform/bento_event_relay
+BENTOV2_EVENT_RELAY_VERSION=latest
 BENTOV2_EVENT_RELAY_CONTAINER_NAME=bentov2-event-relay
 BENTOV2_EVENT_RELAY_INTERNAL_PORT=8080
 BENTOV2_EVENT_RELAY_EXTERNAL_PORT=8750
-BENTOV2_EVENT_RELAY_SOCKET_IO_PATH=/socket.io
+BENTOV2_EVENT_RELAY_SOCKET_IO_PATH=/private/socket.io
```

Comment:
add trailing slash here and I think itll fix one of the bugs with socket.io: `/private/socket.io/`